frontend/src/lib/components/TusUpload.svelte

```diff
@@ -145,6 +145,7 @@
           type="file"
           {accept}
           class="file-input file-input-bordered file-input-primary grow"
+          disabled={percent > 0 && !uploadError}
```

Comment:
That probably works quite well. Though, isn't there a short delay between starting and percent being > 0?

It would make more sense to me to use the explicit status that we're already managing:
```suggestion
          disabled={status === UploadStatus.Uploading || status === UploadStatus.Complete}
```